docs/whatsnew/1.1.0/1_1_0.rst

```diff
@@ -0,0 +1,55 @@
+Pylake 1.1.0
+============
+
+.. only:: html
+
+Pylake `v1.1.0` has been released. Once again, we'll use this section to highlight some of the new features and an important bugfix.
+
+Global analysis
+---------------
+
+Previously, downstream Kymograph analyses were limited to a single kymograph.
+With global analysis, one can combine tracking results from multiple kymographs to get better statistics when estimating dwell times or calculating diffusion constants.
+For more information on this feature, please refer to the relevant section of the :ref:`kymotracking tutorial<global_analysis>`.
+
+.. figure:: global_analysis.png
+
+    Global analysis allows you to analyze multiple kymographs in a single analysis.
+    This results in improved estimates of the biophysical parameters of your experiment.
+
+Bugfix sample_from_image
+------------------------
+
+Prior to version `1.1.0`, the method :meth:`~lumicks.pylake.kymotracker.kymotrack.KymoTrack.sample_from_image` had a bug that assumed the origin of a pixel to be at the edge rather than the center of the pixel.
+Consequently, the sampled window would be off by one pixel half the time.
+Note that the bug does not impact the position estimates.
+
+The old (incorrect) behavior is maintained until the next major release (2.0) to ensure backward compatibility and allow assessment of the impact of the bug.
+For reasonably sized sampling windows (more than 4 pixels wide), the effect of this bug should be small.
+When using :meth:`~lumicks.pylake.kymotracker.kymotrack.KymoTrack.sample_from_image` or :func:`~lumicks.pylake.KymoWidgetGreedy`, it is recommended to include the argument `correct_origin=True` which results in using the correct origin.
+
+Splitting tracks
+----------------
+
+You can now split tracks in the Kymotracker widget.
+
+.. figure:: split.gif
+
+    Clicking with the right mouse button splits an existing line into two segments.
+
+Profile likelihood for dwell time analysis
+------------------------------------------
+
+We added a :ref:`second method<pop_confidence_intervals>` for calculating uncertainty intervals for dwell time analysis.
+This method can be used to determine confidence intervals and assess whether a dwell time model has too many parameters (i.e., if a model with fewer components would be sufficient to fit the data).
+
+.. figure:: pop_ple.png
+
+    Profile likelihood curves for a dwell time model.
+    The intersection points between the blue curve and the dashed lines indicate the 95\% confidence interval for that particular parameter.
+    If the confidence interval for any of the amplitudes contains zero, then that component contributes very little to the model fit and a model with fewer components should be used.
```

Comment:
I think this sentence is a bit too detailed for a "highlights" section, I'd leave it out

Reply:
I assume you mean just the last sentence of the legend? I agree. I'll drop it.

Reply:
yes, I dunno why it marked the whole thing